istio/examples/consumer-provider/Readme.md

```diff
@@ -0,0 +1,70 @@
+# Context
```

Comment:
this "docs/user-guides/integration-istio.md" is imcomplete, as a user my task is not just running a system proccess.  please merge this part to docs/user-guides/integration-istio.md